packages/components/src/translate.js

```diff
@@ -1 +1,12 @@
-export default (key, options) => options.defaultValue;
+import { createInstance } from 'i18next';
+
+// https://github.com/i18next/i18next/issues/936#issuecomment-307550677
+const DEFAULT_I18N = createInstance({}, () => {});
+
+export function getDefaultTranslate(key, options) {
+	return options.defaultValue;
+}
+
+export function getDefaultI18n() {
```

Comment:
`export const DEFAULT_I18N = createInstance({}, () => {});,` 
then 

`import { DEFAULT_I18N } from ...
translate(I18N_DOMAIN_FORMS, { i18n: DEFAULT_I18N })(EnumerationForm);
`

don't work

Reply:
Why ? What is the error ?